content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
++++
+title = "Bevy 0.6"
+date = 2022-01-06
+[extra]
+author = "Carter Anderson"
+twitter = "cart_cart"
+github = "cart"
+youtube = "cartdev"
+image = "bistro_night.png"
+show_image = true
+image_subtitle = "The Lumberyard Bistro scene rendered in the New Bevy Renderer by @mockersf"
+image_subtitle_link = ""
++++
+
+Thanks to **X** contributors, **X** pull requests, and our [**generous sponsors**](https://github.com/sponsors/cart), I'm happy to announce the **Bevy 0.6** release on [crates.io](https://crates.io/crates/bevy)!
+
+For those who don't know, Bevy is a refreshingly simple data-driven game engine built in Rust. You can check out [Quick Start Guide](/learn/book/getting-started/) to get started. Bevy is also free and open source forever! You can grab the full [source code](https://github.com/bevyengine/bevy) on GitHub. Check out [Bevy Assets](https://bevyengine.org/assets) for a collection of community-developed plugins, games, and learning resources.
+
+To update an existing Bevy App or Plugin to **Bevy 0.6**, check out our [0.5 to 0.6 Migration Guide](/learn/book/migration-guides/0.5-0.6/).
+
+There are a _ton_ of improvements in this release, but here are some of the big ones: 
```

Comment:
```suggestion
There are a _ton_ of improvements, bug fixes and quality of life tweaks in this release; here are some of the highlights: 
```

More punch! Less apologetic! Focusing on "we care about the little stuff too" is an important signal of quality.

Reply:
Small style thing: I swapped out the semicolon for a period. I rarely use them :)